lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/service/IdentityVerificationService.java

```diff
@@ -107,7 +106,9 @@ public DocumentCheckVerificationResult verifyIdentity(
                             documentStrengthScore,
                             documentValidityScore,
                             activityHistoryScore);
-                    eventProbe.counterMetric(DCS_CHECK_REQUEST_SUCCEEDED);
```

Comment:
Is this deleted or moved? We don't want dashboards to start missing data

Reply:
To ensure historical data can be viewed going forward the dashboards will need to be updated to add the two sets values.

DCS_CHECK_REQUEST_SUCCEEDED + DOCUMENT_DATA_VERIFICATION_REQUEST_SUCCEEDED.
DCS_CHECK_REQUEST_FAILED + DOCUMENT_DATA_VERIFICATION_REQUEST_FAILED.


```
    // DocumentDataVerification (Request Status)
    public static final String DOCUMENT_DATA_VERIFICATION_REQUEST_SUCCEEDED =
            "document_data_verification_request_succeeded";
    public static final String DOCUMENT_DATA_VERIFICATION_REQUEST_FAILED =
            "document_data_verification_request_failed";
```